client/src/partials/subsidy/subsidy.html

```diff
@@ -0,0 +1,151 @@
+<header data-header>
+  {{ "SUBSIDY.CREATE_TAXE" | translate }}
+</header>
+
+<nav>
+  <div class="pull-left">
+    <ol class="breadcrumb">
+      <li><span class="glyphicon glyphicon-home"></span></li>
+      <li><a href="#/subsidy/">{{"SUBSIDY.TITLE" | translate}}</a></li>
+    </ol>
+  </div>
+  <div class="pull-right">
+    <button class="btn btn-sm btn-default" ng-click="new()">
+      <span class="glyphicon glyphicon-plus"></span>
+       {{ "SUBSIDY.ADD" | translate }}
+    </button>
+  </div>
+</nav>
+
+<main>
+  <div class="row">
+
+    <div class="col-xs-7">
+      <div class="panel panel-primary">
+        <div class="panel-heading">
+          <span class="glyphicon glyphicon-briefcase"></span>
+          {{ "SUBSIDY.REGISTERED" | translate }}
+        </div>
+        <table class="table table-condensed">
+          <thead>
+            <tr>
+              <th>{{ "COLUMNS.LABEL" | translate}}</th>
+              <th>{{ "COLUMNS.VALUE" | translate}}</th>
+              <th>{{ "COLUMNS.IS_PERCENT" | translate}}</th>
```

Comment:
Rather than "is percent" as a column, we should show values like "10%" or "10$".  It is much easier to see, and will free up some more space in this module.